web-console/future-openshift/image-commands/image-commands.md

```diff
@@ -0,0 +1,42 @@
+# Image Commands
+
+## Background
+* Users may need to interact with the OpenShift internal registry directly via the CLI. For example, user may wish to pull an image to debug it locally, or they may want to push an image to create a new Image Stream Tag.
+* Previously, the 3.x console gave users sample CLI commands for interacting with images in OpenShift's internal image registry on Image Stream pages.
+* OpenShift Online also included generic CLI commands on the About page.
+* These commands are useful because they include the location of the internal registry and it is not clear that this information is available elsewhere in the web console.
+* This design makes the commands for interacting with the internal registry available to users through Image Stream and Image Stream Tag pages.
+
+
+## Design
+
+### Image Stream Pages
+
+![image stream prompt](img/image-stream-prompt.png)
+
+* Add the help prompt "Do you need to work with this Image Stream outside of OpenShift?" underneath the **Image Stream Overview** section.
```

Comment:
These designs are solid. I like that they don't clutter up the UI, but are still clearly available enough for those interested.

@ncameronbritt Do you think it's worth calling out "outside of the OpenShift _UI_?" rather than just OpenShift? Major nitpick there but one of the commands exposed is still an OpenShift command.

Reply:
Fair point, @lizsurette. Technically I guess the `oc` command enables a user to work with the image stream outside of OpenShift, but that all seems very pedantic. How about just saying "outside of the web console"?

Reply:
> Fair point, @lizsurette. Technically I guess the `oc` command enables a user to work with the image stream outside of OpenShift, but that all seems very pedantic. How about just saying "outside of the web console"?

+1 to "outside of the web console"...sorry this was SO nit picky hah

Reply:
All good. This is why we have reviews.
![bitmoji](https://render.bitstrips.com/v2/cpanel/9ff55377-6e8f-4a55-9baf-471d1ee75e6a-bc74ffcd-6bb5-4e76-846b-805de97a49b6-v1.png?transparent=1&palette=1&width=246)